### DISCUSSION TITLE
Keep getting error "Id must be a number conforming to the specified constraints"

### DISCUSSION THREAD

**Discussion by hilmanski** [supabase/supabase#7365] [Questions]
HI

When I want to edit a column of my table via GUI
I get this error.
![Screen Shot 2022-06-22 at 09 44 05](https://user-images.githubusercontent.com/4522467/174925429-5b7fb5af-7f4b-4483-8eef-8f4b4838457b.png)

It happens when:
- checking "is_unique" and click save.
or
- adding foreign key to another table

But if I go to terminal to add "UNIQUE" manually, it workds.

**Comment by growthondev** [comment]
it happens to me when I try to remove the foreign key.

**Comment by VladTabacariu** [comment]
same thing here. When editing a column (it's name or type) throws this error

**Comment by davlet61** [comment]
Having the same issue, didn't have it yesterday though

**Reply by davlet61** [reply]
> Having the same issue, didn't have it yesterday though

I ended up deleting the relevant table and creating it manually _(via SQL Editor)_ without a relation. 
My problem was in `public.profiles` table which had an `id` related to `auth.users.id`. 
I am trying to setup Prisma

**Comment by mnpeart** [comment]
Sorry, posted as a reply to previous comment the first time. 

I'm having the same issue and can confirm it was not happening a few hours ago. I also noticed that there are no errors at all in the Postgres logs, which is not surprising given this error on the PATCH request to update a column:

Request URL: `https://api.supabase.io/platform/pg-meta/<project-id>/columns?id=17383.13`
Status: `400`
Request Payload: `{"defaultValue":null,"defaultValueFormat":"expression","id":"17383.13"}`
Response: `{"message":"id must be a number conforming to the specified constraints"}`

When I re-ran the PATCH via curl and changed `id` to `17383` (an int rather than a string) it seems to have worked: 
`'{"defaultValue":null,"defaultValueFormat":"expression","id":17383}'`

**Reply by VladTabacariu** [reply]
Yeah it seems. The number after the decimal represents the index of the column you are trying to edit and the number before decimal it's the table id. So removing that wont work.

**Comment by kavitang** [comment]
This happening for me on any existing column I try to edit via the UI. Everything has to be done manually through the SQL editor, the UI is not allowing me to edit any columns.

**Comment by Jesuscc9** [comment]
Same error here. This might be some kind of issue with the UI, so I will try with the SQL editor

**Comment by d5u** [comment]
I'm getting this error as well. GUI was working yesterday
